doc/scripts/show-versions.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018, Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Out of curiosity, what Apache-2.0 and not BSD-3-Clause like the rest of ACRN source code?